guides/v2.3/extension-dev-guide/declarative-schema/migration-commands.md

```diff
@@ -108,7 +108,7 @@ Each CSV file contains a row that defines the column (or other database entity)
 
 ![Dump Example]({{ page.baseurl }}/extension-dev-guide/declarative-schema/images/dump_example.png)
 
-## Maintain backward compatibility
+## Create a schema whitelist {#create-whitelist}
```

Comment:
There is an industry trend to replace "whitelist" with "allow list" and "blacklist" with block list.  We might want to consider that for our doc style guide.

Reply:
The command name includes the word "whitelist". I can't change it, and it has nothing to do with allowing/blocking,